symfony/translation/5.3/config/packages/translation.yaml

```diff
@@ -0,0 +1,15 @@
+framework:
+    default_locale: en
+    translator:
+        default_path: '%kernel.project_dir%/translations'
+        fallbacks:
+            - en
+#        providers:
```

Comment:
I think most people will only ever use 1 provider, so having something similar to what we have in Mailer as mentioned by Tobia sounds better.

Reply:
Did you mean only 1 exemple provider, like:
```yaml
        providers:
            crowdin:
                dsn: '%env(CROWDIN_DSN)%'
```

If yes, which one do we want to make an example of?

Reply:
More something along the lines of:

```yaml
        provider: '%env(TRANSLATION_PROVIDER_DSN)%'
```

Reply:
Yes, you can add many providers, for example to manage `admin` domain in Loco, and `messages` domain in Crowdin. That's why I added an example for each. But, I'm ok to put only one example under `providers:`

Reply:
I would add 2 but not all, like we do in notifier, but @fabpot please decide, thanks

Reply:
I just commit the removal of 2 providers in the example. Is it ok for you to keep 2 providers here?